Modify playlist

gracewainaina opened the pull request.

Commit:
Edit Profile Activity

Create a Separate activity that allows users to edit their profile

Commit:
Attempt 2: Edit Profile Activity

Create a Separate activity that allows users to edit their profile

Commit:
Set up Profile Photo and update music bio

-The user can set up a profile photo by either taking a picture using a camera or by uploading a picture from the gallery
-The user can also updated their music bio
-All these happens in the edit profile activity

Commit:
Update profile demo

-The demo shows the user setting a profile picture using both camera and gallery
-Also shows user updating music bio

Commit:
Set up playlist icon

The user can set up a playlist icon by either taking a picture using the camera or uploading from the gallery

Commit:
Populate newly created playlist

User can now add a song to a newly created playlist

xuyueee commented:
@gracewainaina This video seems to be showing how to update the user's profile, not updating the playlist.

Commit:
Play song, login error handling, contant string extras for intents

- After creating/ joining playlist, user can add songs to the shared playlist and play each song at a time.
- Added error handling code for unsuccessful login to both Musicca and Spotify
- Made string extras for intents constants in each fragment and activity

Commit:
Demo for play song after joining/ creating a playlist

- user can add and play songs to a playlist they created or joined
